packages/11ty/helpers/page-filters.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * Returns `true` if page should be built
+ * for the current output type defined in `env.QUIRE_OUTPUT`
+ * @param  {Object} data Eleventy `page.data` object
+ * @return {Boolean}
+ */
+const buildFilter = function ({ outputs }) {
```

Comment:
@hbalenda `buildFilter` might be a method that _builds_ a filter (which this method does, however the other two methods are named for the things the filter function returns) should this method be named `outputsFilter`?

Reply:
hrmm true, `currentOutputFilter` ?